AmahiAnywhere/Podfile.lock

```diff
@@ -24,4 +24,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 016a36dd306797a6f49fe648c4298e22ce2a2570
 
-COCOAPODS: 1.4.0
+COCOAPODS: 1.4.0.rc.1
```

Comment:
I think it is better we stick to Stable releases. The rc.1 is beta

Reply:
Okay I will update this